app/models/bot/smooch.rb

```diff
@@ -430,7 +430,7 @@ def self.get_custom_menu_options(state, workflow, uid)
         self.get_supported_languages.each do |l|
           i = self.get_next_menu_item_number(i)
           options << {
-            'smooch_menu_option_keyword' => [l, i].join(','),
+            'smooch_menu_option_keyword' => i.to_s,
```

Comment:
Do not add the language code as a valid key for a tipline menu language option. Now the key is just a number, which is the order of that option in the menu. This was supposed to serve as shortcuts for language codes but there are edge cases (for example, "hi" can be misinterpreted as a shortcut for "Hindi", if the tipline supports that language).